src/mame/heathkit/tlb.cpp

```diff
@@ -709,6 +720,41 @@ static INPUT_PORTS_START( watz19 )
 
 INPUT_PORTS_END
 
+static INPUT_PORTS_START( gp19 )
+	PORT_INCLUDE( tlb )
+
+	PORT_MODIFY("SW402")
+	PORT_DIPNAME( 0x80, 0x00, "Automatic Holdscreen")    PORT_DIPLOCATION("SW402:8")
+	PORT_DIPSETTING(    0x00, "Disabled (VT100 mode)")
+	PORT_DIPSETTING(    0x80, "Enabled (Z19 mode)")
+
+	PORT_START("SW1")
+	PORT_DIPNAME( 0x03, 0x01, "Trailing characters for Tektronix message")   PORT_DIPLOCATION("SW1:1,2")
+	PORT_DIPSETTING(    0x00, "none")
+	PORT_DIPSETTING(    0x01, "CR")
+	PORT_DIPSETTING(    0x02, "none")
+	PORT_DIPSETTING(    0x03, "CR,EOT")
```

Comment:
Is there a reason for not capitalising the first letter of “none” in the settings here, when you’ve capitalised the first letter of other settings that don’t begin with a proper noun or initialism (e.g. “Normal”, “Never”)?

Reply:
No real reason, should these use `DEF_STR(None)`, I wasn't sure what value that had vs. just used `"None"`